com.arc.embeddedcdt/src/com/arc/embeddedcdt/dsf/utils/Configuration.java

```diff
@@ -48,10 +48,24 @@ private static boolean getAttribute(ILaunchConfiguration lc, String attribute, b
         }
     }
 
+    private static int getAttribute(ILaunchConfiguration lc, String attribute, int defaultValue) {
+        try {
+            return lc.getAttribute(attribute, defaultValue);
+        } catch (CoreException e) {
+            e.printStackTrace();
+            return defaultValue;
+        }
+    }
+
     public static String getProgramName(ILaunchConfiguration lc) {
         return getAttribute(lc, ICDTLaunchConfigurationConstants.ATTR_PROGRAM_NAME, "");
     }
 
+    public static int getDebuggerPluginVersion(ILaunchConfiguration lc){
+        return getAttribute(lc, LaunchConfigurationConstants.ATTR_DEBUGGER_PLUGIN_VERSION,
+                LaunchConfigurationConstants.UNREAL_DEBUGGER_PLUGIN_VERSION_NUMBER);
+    }
+
```

Comment:
In retrospect I'm not happy with this class in general - it only reads properties, but doesn't set them. It also shouldn't be "static". Instead it should act as a wrapper - one creates instance of this "Configuration" class and passes ILaunchConfiguration as an argument to constructor and it is stored as a field. Then all of those methods will not need to take `lc` as an argument. And `set` methods should be added. That would allow to move most of the `Constants` fields to become private fields of this `Configuration` file. You don't need to do that for this pull request, but can refactor the code after this change will be merged.